submissions/AndewTu/js-dom/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+  	<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1, shrink-to-fit=no">
+  	<link rel="stylesheet" href="app.css">
+<title>Paw patrol</title>
+</head>
+<body>
+	<div class="content">
+	<div id="main" class="container">
+		<div class="navi"> 
+			<ul>
+				<li><button class ="btn" id="marshall">Marshall</button></li>
+				<li><button class ="btn" id="chase">Chase</button></li>
+				<li><button class ="btn active" id="skye">Skye</button></li>
+				<li><button class ="btn" id="rubble">Rubble</button></li>
+				<li><button class ="btn" id="rocky">Rocky</button></li>
+				<li><button class ="btn" id="zuma">Zuma</button></li>
+				<li><button class ="btn" id="everest">Everest</button></li>
+			</ul>
+		</div>
+		<div class="puppies">
+				<h1 class="name">Skye</h1>
+				<img class="pic" src="img/skye.png" caption="puppy">
+				<p class="desc">At seven years old, Skye is a cute, smart Cockapoo puppy. She is a fearless daredevil who will try anything with grace and a smile. She’s smart, loyal, and quick with a quip, sometimes gently teasing the bigger dogs who can’t seem to keep up! She loves to get paw-dicures and look good. When there’s no mission for the PAW Patrol, she can be found snowboarding or playing her favorite videogame, Pup Pup Boogie.</p>
+		</div>	
+	</div>
+	</div>
+<script src="app.js"></script>	
+</body>
+</html>
```

Comment:
It's good practice, to leave an empty line after your code. This task can be automated by your IDE.